### DISCUSSION TITLE
Welcome to Algorithms Discussions!

### DISCUSSION THREAD

**Discussion by Kumar-laxmi** [Kumar-laxmi/Algorithms#296] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kaushan-Dutta** [comment]
Hi everyone!! I am Kaushan Dutta and I am currently a 2nd Year student pursuing his BTech in CSE. I am currently working on Full stack BlockChain Development . I have been a MERN Stack Developer and have participated in a number of hackathons like Hackobertfest, HackOdisha, Amritathon and so on. I am looking forward to contribute as much in this event and love to work with everyone of you.
Connect me :
Website: https://kaushan-dutta.vercel.app/
Linkedin: https://www.linkedin.com/in/kaushan-dutta-bb68b021a/
